templates/pages/self-service/service_catalog.html.twig

```diff
@@ -82,11 +110,14 @@
     </section>
 
     <script>
+
         (async function() {
             const module = await import(
                 "{{ js_path('js/modules/Forms/ServiceCatalogController.js') }}"
             );
-            new module.GlpiFormServiceCatalogController();
+            window.GlpiServiceCatalog = new module.GlpiFormServiceCatalogController(
+                {{ icons|json_encode|raw }}
+            );
```

Comment:
I would prefer something like this, I think it is safer for XSS:

```suggestion
            const icons = {};
            {% for key, icon in icons %}
                icons["{{ key.escape('js') }}"] = "{{ icon|escape('js') }}";
            {% endfor %}
            window.GlpiServiceCatalog = new module.GlpiFormServiceCatalogController(icons);
```

I didn't test it so be careful but the idea is to benefit from twig escaping and avoid copying raw json into the javascript code.